cmd/nanobus/main.go

```diff
@@ -142,11 +134,10 @@ func (c *runCmd) Run() error {
 }
 
 type invokeCmd struct {
-	DeveloperMode bool `name:"developer-mode" help:"Enables developer mode."`
+	Target        string `arg:"positional" required:"" help:"The application configuration yaml file or OCI image reference."`
```

Comment:
This one is a bit trickier as the common command will be `nanobus invoke [operation]`. I can't see a way to make both `nanobus invoke [operation]` _and_ `nanobus invoke <target> [operation]` work. This means we probably need to make `target` a flag in this case :/

I'll keep looking at the options [Kong](https://github.com/alecthomas/kong) provides but for now this will keep our examples working.

```
`name:"target" default:"bus.yaml" help:"The application configuration yaml file or OCI image reference."`
```